templates/layouts/configs/_feedback.html

```diff
@@ -9,10 +9,10 @@
     },
     "mailto": {
       "address": "census2021.feedback@ons.gov.uk",
-      "textBefore": _("Tell us what you think of this website by emailing"),
-      "textAfter": _("This email is for feedback only and we will not be able to reply. Do not include any personal details."),
+      "textBefore": _("Give us feedback and help us improve this service.<br>Tell us what you think of this service by emailing"),
```

Comment:
This will repeat the duplicate the text "Give us feedback and help us improve this service." immediately beneath the dropdown link containing the same text - doesn't look like this is what the card is asking for?

Reply:
Yeah you're right ill remove that